metadata/algolia/jobs.xml

```diff
@@ -114,4 +114,31 @@ Can perform full records updates or partial records updates. See the 'indexingMe
         <triggers/>
     </job>
 
+    <job job-id="AlgoliaContentIndex_v2" priority="0">
+        <description>Index all searchable content assets assigned to the selected site to Algolia.</description>
+        <parameters/>
+        <flow>
+            <context site-id="RefArch"/>
+            <step step-id="SFCCContentSearchIndex" type="SearchReindex" enforce-restart="false">
+                <description>To ensure that we can rely on SFCC search indexing, it is recommended to reindex all search content indexes. This will help improve the accuracy and relevance of the search results on your website.</description>
+                <parameters>
+                    <parameter name="Product related search indexes">false</parameter>
+                    <parameter name="Active data search index">false</parameter>
+                    <parameter name="Content search index">true</parameter>
+                    <parameter name="Indexer Type">Full Index Rebuild</parameter>
+                </parameters>
+            </step>
+            <step step-id="algoliaContentIndex" type="custom.algoliaContentIndex" enforce-restart="false">
+                <description>Index all searchable contents assigned to the selected site to Algolia. The list of indexed attributes is configurable.
+It highly depends on the content search index on Salesforce, so do not forget to trigger search index after content update or enable incremental update.</description>
```

Comment:
Since you've added a step that systematically rebuild the index, we can remove the second part:
```suggestion
                <description>Index all searchable contents assigned to the selected site to Algolia. The list of indexed attributes is configurable.</description>
```